### ISSUE TITLE
ExampleMod : ExampleChandeliers Should Be Provide

### ISSUE THREAD

**Issue by Omnielement** [bluemagic123/tModLoader#3460]
### Do you intend to personally contribute/program this feature?

Yes

### I would like to see this change made to improve my experience with

Mod capability as a Modder

### Description

I think ExampleMod needs to provide an ExampleChandeliers just like Chandeliers in 1.4 Terraria.
*It will tilt in the wind but keep silence in front of the wall.
*It will swing violently when players swept through it swiftly.
*It also needs to suit with colorful paints.
*It may vary in physical structure, like sandstone chandelier, copper chandelier, crystal chandelier, and stardust chandelier.

I have tried to make modChandeleier by myself, but it can't suit with paints.(Referring to dfgasdfg/Everglow/Sources/Modules/Myth/TheFirefly/Tiles/Furnitures/GlowWoodChandelier)

### What does this proposal attempt to solve or improve?

Making a mod chandelier is really hard.This can also standardize modders making chandeliers. 

### Which (other) solutions should be considered?

Mod vines, swing like 1.4Terraria.
According to the vines behaved physical-badly  in vanilla, this should be take more consider.
(When the vine too long, you can see an unforgettable sight)

**Comment by Omnielement** [comment]
![Chandeliers](https://user-images.githubusercontent.com/103014909/236686373-59ce414c-d502-4338-823f-c433745f498d.gif)

**Comment by Omnielement** [comment]
![Chandeliers](https://user-images.githubusercontent.com/103014909/236686495-dbc35ee3-23c0-473a-8d18-b926386d8f21.gif)
This gif shows my chandeliers.

**Comment by JavidPack** [comment]
I'll revisit this when implementing #3442

**Comment by JavidPack** [comment]
So, it's not a chandelier, but in https://github.com/tModLoader/tModLoader/pull/4429 I updated `ExampleAnimatedTile`/"Red Firefly in a Bottle" to support swaying. It is a lantern, 1x2, but the code should work all the same for a chandelier tile with no modifications. ExampleLamp has the code for toggling state to turn off the lights.

Closing this now, but we wouldn't reject an ExampleChandelier if anyone wanted to submit one sometime.
